lang/gcc10-bootstrap/Portfile

```diff
@@ -176,20 +180,25 @@ if {${os.major} >= 18 && ${configure.sdkroot} ne ""} {
     configure.args-append --with-sysroot="[regsub {MacOSX1[0-9]\.[0-9]+\.sdk} ${configure.sdkroot} {MacOSX.sdk}]"
 }
 
-# clang (as) from Xcode 12.5 has various problems with gcc build
-if { ${os.platform} eq "darwin" && \
-         ( [ vercmp ${xcodeversion} 12.5 ] >= 0 || [ vercmp ${cltversion} 12.5 ] >= 0 ) } {
-    pre-configure {
-        ui_warn "Applying '--without-build-config' workaround to Xcode ${xcodeversion} / CLT ${cltversion}"
-        ui_warn "If versions > 12.5 please check if it is still required"
-    }
-    # gcc has build issues on macOS 11.3 with the use of Xcode clang as 'as'
-    # https://gcc.gnu.org/bugzilla/show_bug.cgi?id=100340
-    # https://trac.macports.org/ticket/62775
-    configure.args-append  --without-build-config
+if {${os.major} < 18 || ${os.major} > 19} {
+    default_variants    +universal
 }
 
-default_variants    +universal
+platform powerpc {
+    configure.universal_archs ppc ppc64
+}
+if { ${os.platform} eq "darwin" && ${os.major} >= 20 } {
+    platform i386 {
+        configure.universal_archs x86_64 arm64
+    }
+    platform arm {
+        configure.universal_archs x86_64 arm64
+    }
+} else {
+    platform i386 {
+        configure.universal_archs i386 x86_64
+    }
+}
```

Comment:
I'm confusing about this block. Any reason when `configure.universal_archs` may have different value from which you have put here.

Reply:
I’d borrow led this for gcc-devel?

Later versions of macOS will still say there an i386 platform when when they only run x86_64. This would account for this.

Reply:
I had it here before, but for some reason had removed it. Let me try to recall why.

Reply:
I’ll just blame me for that one idk lol